src/components/AssetManagementPanel/AssetManagementForm/FormVariants/SurrenderForm/SurrenderForm.tsx

```diff
@@ -35,6 +40,22 @@ export const SurrenderForm = ({
     onSetFormAction(AssetManagementActions.None);
   };
 
+  const { setOverlayContent, setOverlayVisible } = useContext(OverlayContext);
```

Comment:
how about a function `showOverlay` that does both setting of the content and making it visible? so that we dont have to fire the two function separately all the time? Applies to the component above also

Reply:
removed setting visibility individually on user click, setting visibility on callback in AppContainer.js instead.

![Screenshot 2020-05-05 at 10 40 13 PM](https://user-images.githubusercontent.com/4774314/81078935-6f2af380-8f21-11ea-9c9c-a853574b9393.png)